CONTRIBUTING.md

```diff
@@ -113,6 +116,13 @@ Follow the installation instructions here: [task-install](https://taskfile.dev/i
    - **Windows:**
      - Install `mingw` by following the instructions from [mingw-w64.org](https://www.mingw-w64.org/downloads) to provide the necessary `gcc` compiler for building MassaStation.
 
+     We recommend to install in a Powershell as an administrator:
```

Comment:
Same as before.

Also, since we have a task command to install some dependencies such as `stringer` and `go swagger` (https://github.com/massalabs/station/blob/a7a4ed04d3751d194a8161d914010c03231d6104/Taskfile.yml#L7C1-L8C63), shouldn't we just add another task such as "install-lint", or just add those installs to the `install` command ?